### PR TITLE
add initial sources and specs

### PR DESCRIPTION
**Pull request by benjamreis**

### CODE REVIEW

File: SPECS/security-tools.spec [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Name:           security-tools
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        Generates 3 UUIds for a pool secret
```
Comment (Wescoeur) [inline SPECS/security-tools.spec]: `uuids` or `UUIDs`

File: SPECS/security-tools.spec [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+cd build
+make install
+
+%files
+%doc LICENSE README.md
+/usr/bin/pool_secret
+
+%changelog
+* Thu May 02 2019 Benjamin Reis <benjamin.reis@vates.fr> - 1.0.0-1
+- Initial package
```
Comment (Wescoeur) [inline SPECS/security-tools.spec]: Missing NL.

File: SPECS/security-tools.spec [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+%description
+Generates 3 UUIds for a pool secret.
+
+%prep
+%autosetup -p1
+
+%build
+mkdir build && cd build
+%cmake3 .. -DCMAKE_INSTALL_PREFIX=%{buildroot}/usr/bin
```
Comment (Wescoeur) [inline SPECS/security-tools.spec]: `CMAKE_INSTALL_PREFIX` is just the installation prefix of any file. The default value is `/usr/local`, you must set it to `%{buildroot}/usr` here.

See: https://github.com/xcp-ng/security-tools/blob/master/CMakeLists.txt#L72
`CMAKE_INSTALL_BINDIR` is the concatenation of `CMAKE_INSTALL_PREFIX` with `bin`.

File: SPECS/security-tools.spec [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+License:        GPLv3
+URL:            https://github.com/xcp-ng/security-tools
+Source0:        https://github.com/xcp-ng/security-tools/archive/v%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:  cmake3
+BuildRequires:  make
+BuildRequires:  gcc
+
+%description
+Generates 3 UUIds for a pool secret.
```
Comment (Wescoeur) [inline SPECS/security-tools.spec]: Same as above.

File: SPECS/security-tools.spec [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+Release:        1%{?dist}
+Summary:        Generates 3 uuids for a pool secret
+License:        GPLv3
+URL:            https://github.com/xcp-ng/security-tools
+Source0:        https://github.com/xcp-ng/security-tools/archive/v%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:  cmake3
+BuildRequires:  make
+BuildRequires:  gcc
+
```
Comment (Wescoeur) [inline SPECS/security-tools.spec:13]: Add a `BuildRequires:  openssl-libs` because the binary is linked with `-ldl`.

**Comment by Wescoeur** [comment]
You can use koji to validate the RPM and paste the result link here. :)

**Comment by benjamreis** [comment]
https://koji.xcp-ng.org//taskinfo?taskID=5481

**Pull request merged**